.github/CODEOWNERS

```diff
@@ -400,8 +400,118 @@
+/x-pack/plugins/security_solution/cypress/integration/pagination @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/screens/hosts @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/screens/network @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/tasks/hosts @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/tasks/network @elastic/security-threat-hunting-explore
+
+/x-pack/plugins/security_solution/public/common/components/charts @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/header_page @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/header_section @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/hover_actions @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/hover_actions/actions @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/inspect @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/last_event_time @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/links @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/matrix_histogram @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/navigation @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/news_feed @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/overview_description_list @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/paginated_table @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/sidebar_header @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/stat_items @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/tables @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/top_n @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/with_hover_actions @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/containers/hosts_risk @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/containers/matrix_histogram @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/lib/cell_actions @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/cases @elastic/security-threat-hunting-explore
```

Comment:
Should this still be explore or the `response ops cases` team?

Reply:
Based on the comment from @cnasikas this is now Explore ownership https://github.com/elastic/kibana/pull/128218#discussion_r848350332